packages/st2/debian/st2.links

```diff
@@ -12,3 +13,6 @@ opt/stackstorm/st2/bin/st2-track-result usr/bin/st2-track-result
 opt/stackstorm/st2/bin/st2-validate-pack-config usr/bin/st2-validate-pack-config
 opt/stackstorm/st2/bin/st2-check-license usr/bin/st2-check-license
 opt/stackstorm/st2/bin/st2ctl usr/bin/st2ctl
+
+
+
```

Comment:
Can we remove these extra unnecessary spaces?

Reply:
Ok, Can you point to why this would be treated as file and not an executable:
File "/usr/bin/st2-apply-rbac-definitions"
        behaves like script or binary
          is expected to be file and be executable (FAILED - 1)

Reply:
The instructions to ship the `/usr/bin/st2-apply-rbac-definitions` via packaging we add here should fail before we merge the st2 RBAC PR: https://github.com/StackStorm/st2/pull/5086.
Once the st2 PR is merged the packages CI build should change its status to :heavy_check_mark: and can be merged too to finalize the RBAC integration.